### PR TITLE
z_sound_source OK

### PR DESCRIPTION
**Pull request by engineer124**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

**Comment by engineer124** [comment]
Converting this to draft as there is an active discussion in OoT about naming these functions. Will name these functions once a consensus is reached on naming.

**Comment by engineer124** [comment]
Undrafting this as there is "more-or-less" consensus on these filenames in OoT, so the names here parallel those with the exception of the new function that gives the option to play the sfx again every frame for the duration of the request.

File: src/code/audio/code_801A5BD0.c [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/code_801A5BD0/func_801A7284.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_801A5BD0/func_801A72CC.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/code_801A5BD0/Audio_StopSfxByPos.s")
```
Comment (EllipticEllipsis) [inline src/code/audio/code_801A5BD0.c:27]: I still don't like this name much, I think `From` or `At` work better.

**Review by EllipticEllipsis — APPROVED** [review]
I have one comment, that we can argue about in a subsequent audio PR, so just going to merge it as is

File: src/code/z_sound_source.c [hunk 2]
```diff
@@ -0,0 +1,75 @@
+#include "global.h"
```
Comment (EllipticEllipsis) [inline src/code/z_sound_source.c:1]: Oh, I suppose this is also missing a file description, but I imagine the whole lot is going to get a second pass anyway.

**Pull request merged**